M2/Macaulay2/packages/MatrixSchubert/permutationMethods.m2

```diff
@@ -226,10 +226,14 @@ isPatternAvoiding (List,List) := Boolean => (perm, pattern) -> (
     --assume permutation is pattern-avoiding, break if not true
     isAvoiding := true;
     for idx in subsets(0..#perm-1, #pattern) do {
-        sortedIdx := sort(idx);
-        pairwiseComparison := apply(pattern_{0..#pattern-2}, pattern_{1..#pattern-1}, (i,j) -> perm#(sortedIdx#(i-1)) < perm#(sortedIdx#(j-1))); -- pairwise comparison of permutation according to pattern
-        isAvoiding = not all(pairwiseComparison, i -> i == true); -- true if there was one inequality that failed, else all inequalities are true and so not pattern-avoiding
-        if not isAvoiding then break;
+        vals := perm_(sort idx);
```

Comment:
I'm almost certain that `idx` will already be sorted, so this call to `sort` is likely redundant.